GCC_2019_07_10/dbplyr lightning talk/03 linking example.R

```diff
@@ -27,23 +27,24 @@ simple_SMR01 <- tbl(SMRA_connection, "SMR01_PI") %>%
 # Do a link using dplyr joins
 # Alternative is (getting) complicated SQL
 # Or extract separately and then join which is to be avoided!
-linked_1 <- simple_SMR01 %>% inner_join(deaths)
+linked_1_query <- simple_SMR01 %>% inner_join(deaths)
 
 # We could probably write some slightly better SQL but not as easily!
-linked_1 %>% show_query()
+linked_1_query %>% show_query()
 
 # Do the extract
-linked_1 <- collect(linked_1)
+linked_1_extract <- collect(linked_1_query)
 
 # The larger the extracts the faster this method is compared to joining separate extracts
 
 # This is easily extended to multiple links, for example by including SMR04
 simple_SMR04 <- tbl(SMRA_connection, "SMR04_PI") %>%
   select(LINK_NO, ADMISSION_DATE, DISCHARGE_DATE) %>%
-  filter(DISCHARGE_DATE >= To_date("2019-05-01", "YYYY-MM-DD")) # Had to go one month earlier!
+  # Pick a recent date for the example
+  filter(DISCHARGE_DATE >= To_date("2020-05-01", "YYYY-MM-DD"))
 
 # Do a full join on the SMR01 and 04 data
-linked_2 <- simple_SMR01 %>%
+linked_2_extract <- simple_SMR01 %>%
```

Comment:
Is linked_2_extract a second example of a linked extract? (Sorry I don't understand this on my own).
If it is the second example, then can it be made clear in the comments above it OR change the name of the variable, for ex.:
ex_1_simple_query
ex_1_simple_extract
ex_2_simple_extract?

Reply:
Haha, this was the bit that was supposed to make it clearer. I'll have a think!